.github/workflows/pytest.yml

```diff
@@ -161,7 +161,8 @@ jobs:
           path: .coverage
 
   ci-integration:
-    runs-on: ubuntu-latest-8core
+    # runs-on: ubuntu-latest-8core
+    runs-on: blacksmith-4vcpu-ubuntu-2404
```

Comment:
It wasn't *quite* a one-line change, but close.

Reply:
Let's chuck a comment in here explaining the difference between ubuntu and blacksmith runners.